clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1447,13 +1447,13 @@ def VTableAddrPointOp : CIR_Op<"vtable.address_point",
 
 def SetBitfieldOp : CIR_Op<"set_bitfield"> {
   let summary = "Set a bitfield";
-  let description = [{ 
```

Comment:
I don't see these trailing whitespaces in the repo, since it's a bit distracting, it would be nice to update the PR to remove them.

Reply:
Please remove the whitespace changes from this PR.